examples/todo-list/src/index.ts

```diff
@@ -12,8 +12,7 @@ export async function main(options?: ApplicationConfig) {
   await app.boot();
   await app.start();
 
-  const server = await app.getServer(RestServer);
-  const port = await server.get<number>('rest.port');
-  console.log(`Server is running at http://127.0.0.1:${port}`);
+  const url = app.restServer.url;
```

Comment:
can we remove the `RestServer` import as well?

Reply:
Hmm, unused imports aren't being flagged again it seems :(